client/src/pages/ClassList.jsx

```diff
@@ -106,6 +106,16 @@ class ClassList extends Component {
                 accessor: 'capacity',
                 filterable: true,
             },
+            {
+                Header: 'Max Virtual Capacity',
+                accessor: 'maxCapacity',
```

Comment:
Does it matter if the `accessor` field is duplicated with the one from `maxCapacity`?